src/app/components/tree-view/index.html

```diff
@@ -39,7 +43,7 @@ <h3 class="sky-text-success">Use when</h3>
     <sky-demo-page-content sectionHeading="Anatomy">
       <sky-docs-anatomy imagePath="~/assets/img/guidelines/angular-tree-view/anatomy.png">
         <sky-docs-anatomy-item anatomyNumber="1">Expand/collapse button</sky-docs-anatomy-item>
-        <sky-docs-anatomy-item anatomyNumber="2">List item content</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="2">List item</sky-docs-anatomy-item>
         <sky-docs-anatomy-item anatomyNumber="3" isOptional="true">Checkbox</sky-docs-anatomy-item>
         <sky-docs-anatomy-item anatomyNumber="4" isOptional="true">Context menu</sky-docs-anatomy-item>
```

Comment:
Should we really call out context menus here? And if so, should we include an entry in the Options section? We highlight context menus in the demo, but as far as I can tell, this is not an option of our components and is something that consumers need to set on the Angular component we are building off of. So it seems like maybe this shouldn't really be here. But if we list cascading as an anatomy item and in the demo, then shouldn’t we also call out this option in the Options section?

Reply:
For the UX pattern, context menus are an option that the designer needs to consider. If we don't include them in the example, designers will think they can't use them. Need to think about whether to include it in the Options section.

I am not sure how the consumer technically includes them.

Reply:
Makes sense. Thanks!